src/client/components/SliderDotsComponent/SliderDotsComponent.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './SliderDotsComponent.styles.css';
+
+export default function SliderDotsComponent(props) {
+  return (
```

Comment:
This is not that bad, but it still needs improvement. Variable names like image, i and e should be avoided. Also prefer separating the logic from the view.
Like this:
```
 {props.activeImageIndex === i ? (
            <circle cx="4" cy="4" r="4" fill="#795548" />
          ) : (
            <circle cx="4" cy="4" r="4" fill="gray" />
          )}
```
Only the author knows what this means. I suggest extracting it into a well named function

Reply:
Sorry lets just leave it. It's pretty obvious. Just the duplication that confused my initially

Reply:
:)

Reply:
Ok no problem. I will then go ahead and resolve this conversation.